dead.py

```diff
@@ -23,29 +24,38 @@
 TYPE_IGNORE_RE = re.compile(TYPE_COMMENT_RE.pattern + r'ignore\s*(#|$)')
 # https://github.com/python/typed_ast/blob/55420396/ast27/Grammar/Grammar#L147
 TYPE_FUNC_RE = re.compile(r'^(\(.*?\))\s*->\s*(.*)$')
+DISABLE_COMMENT_RE = re.compile(r'#.*dead\s*:\s*disable')
 
 
 class Visitor(ast.NodeVisitor):
     def __init__(self) -> None:
         self.filename = ''
+        self.lines: List[str] = []
         self.is_test = False
         self.reads: UsageMap = collections.defaultdict(set)
         self.defines: UsageMap = collections.defaultdict(set)
         self.reads_tests: UsageMap = collections.defaultdict(set)
 
+    def disabled(self, node: ast.AST) -> bool:
+        line = self.lines[node.lineno - 1]
+        return bool(DISABLE_COMMENT_RE.search(line))
```

Comment:
I think it should still identify all things that are unused, but at the end filter them out?  that way the information can still be retained